### DISCUSSION TITLE
Weder eine Erweiterung noch ein Austausch ist in dieser Geräteklasse möglich.

### DISCUSSION THREAD

**Discussion by GiK90** [camueller/SmartApplianceEnabler#406] [Q&A]
Ich habe alle Geräte bisher ohne Probleme einrichten können.
Nun zeigt es mir immer oben genannte Fehlermeldung an.

<img width="889" alt="Bildschirm­foto 2022-12-21 um 12 56 33" src="https://user-images.githubusercontent.com/62926420/208899833-5c9b5e3b-74a5-4c1b-8b62-1e2cb9c84a5f.png">

Ich messe aktuell 10 Geräte über den SAE und das wäre das 11. Was mache ich hier falsch?

Danke!

**Comment by camueller** [comment]
Hast Du vieleicht dieselbe ID zweimal verwendet? Ich habe 22 Geräte registriert im Sunny Portal.

**Comment by chubakka0815** [comment]
Ich habe leider seit ein paar Tagen dasselbe Problem, das ich keine neuen Geräte im Sunny Portal hinzufügen kann. Hab auch noch keine Idee warum das so ist. Hatte mir noch zusätzlich vier Shelly Plug S zugelegt, da die anderen gut für meine Zwecke funktionieren. SAE hatte ich letzte Woche auf 2.1.0 aktualisiert.

**Comment by camueller** [comment]
Bei einem Problem mit dem Sunny Portal muss man sich an den Support von SMA wenden. Hier wird wohl niemand weiterhelfen können. Das Problem tritt nicht auf, weil Ihr kürzlich den SAE auf 2.1.0 aktualisiert habt, sondern weil ihr überhaupt Geräte hinzufügen wollt.

**Comment by chubakka0815** [comment]
Sorry, war mein Fehler. Hatte 12 Geräte registriert und war somit an der "ersten" Grenze angekommen. Funktioniert bei mir also einwandfrei
